modules/ROOT/pages/manage-proxy.adoc

```diff
@@ -7,40 +7,53 @@ This topic describes how to set or update proxy settings used by Runtime Fabric
 
 == Update the Monitoring Proxy
 
-. Obtain the https://runtime-fabric.s3.amazonaws.com/rtfctl/rtfctl-0.2.0[latest version] of `rtfctl` and transfer to a controller node under the `/usr/local/bin` folder with the name `rtfctl`. You may need root access to perform this operation.
+. Obtain the https://runtime-fabric.s3.amazonaws.com/rtfctl/rtfctl-0.2.30[latest version] of `rtfctl` and transfer to a controller node under the `/usr/local/bin` folder with the name `rtfctl`. You may need root access to perform this operation.
 . Make the script executable by running `chmod +x /usr/bin/rtfctl`.
 . On the controller node, run `rtfctl -h` to verify the script works as expected by outputting instructions on how to use the utility.
 . Run the following command, replacing the placeholder values with the following:
 +
 ----
-rtfctl apply monitoring-proxy "<user>:<pass>@<10.0.0.2>:<8080>"
+rtfctl apply monitoring-proxy "socks5://<user>:<pass>@<10.0.0.2>:<8080>"
 ----
 +
 .. *<user>*: the username needed to authenticate to the SOCKS5 proxy.
 .. *<pass>*: the password needed to authenticate to the SOCKS5 proxy.
 .. *<10.0.0.2>*: the IP address or hostname to access the SOCKS5 proxy.
 .. *<8080>*: the port on the host where the SOCKS5 proxy is listening for requests.
-. To verify the change was successful, run the following command to output the current value of the monitoring proxy: `rtfctl get monitoring_proxy`
+. To verify the change was successful, run the following command to output the current value of the monitoring proxy: `rtfctl get monitoring-proxy`
 
 The output should match the value expected.
 
 == Update the HTTP Proxy
 
-. Obtain the xref:https://runtime-fabric.s3.amazonaws.com/rtfctl/rtfctl-0.2.0[latest version] of `rtfctl` and transfer to a controller node under the `/usr/local/bin` folder with the name `rtfctl`. You may need root access to perform this operation.
+This operation requires two procedures: running a script on each node of your Runtime Fabric to update the cluster, and applying the proxy values to the services running within the Runtime Fabric.
+
+=== Run script on each node
+
+. Copy the `set_dockerd_proxy.sh` from the controller node used to install Runtime Fabric located under `/opt/anypoint/runtimefabric/scripts/` to each controller and worker node under the same location (`/opt/anypoint/runtimefabric/scripts/`).
```

Comment:
should we assume that they have the latest installer here?

Reply:
Good point. We'll need to provide this script in a location (similar to the Mule license key update script) for customers running on older versions to obtain.

Reply:
Added a note to guide customers to open a support ticket if unable to locate the script.

Reply:
👍